chain/client/tests/challenges.rs

```diff
@@ -540,17 +540,18 @@ fn test_verify_chunk_invalid_state_challenge() {
             challenge_body.partial_state.0,
             vec![
                 vec![
-                    1, 7, 0, 20, 155, 199, 55, 40, 218, 150, 222, 64, 132, 213, 252, 78, 132, 13,
-                    31, 108, 106, 36, 32, 241, 213, 207, 255, 230, 98, 36, 34, 59, 131, 51, 40, 83,
-                    252, 63, 177, 215, 80, 204, 201, 233, 89, 151, 192, 80, 3, 13, 123, 166, 78,
-                    235, 195, 174, 220, 16, 53, 121, 47, 85, 152, 199, 25, 129, 208, 171, 30, 7,
-                    228, 175, 99, 17, 113, 5, 94, 136, 200, 39, 136, 37, 110, 166, 241, 148, 128,
-                    55, 131, 173, 97, 98, 201, 68, 82, 244, 223, 70, 86, 83, 135, 2, 0, 0, 0, 0, 0
+                    1, 7, 0, 49, 175, 194, 193, 175, 131, 148, 68, 194, 63, 147, 77, 57, 224, 141,
+                    233, 242, 68, 18, 172, 251, 142, 170, 231, 194, 179, 155, 39, 7, 72, 67, 13,
+                    125, 255, 115, 248, 177, 217, 13, 101, 215, 126, 200, 235, 63, 88, 166, 210,
+                    152, 205, 174, 37, 238, 67, 244, 106, 68, 57, 151, 23, 90, 167, 66, 76, 171,
+                    30, 7, 228, 175, 99, 17, 113, 5, 94, 136, 200, 39, 136, 37, 110, 166, 241, 148,
+                    128, 55, 131, 173, 97, 98, 201, 68, 82, 244, 223, 70, 86, 137, 137, 2, 0, 0, 0,
+                    0, 0
```

Comment:
the test relies on `runtime/near-vm-runner/tests/res/test_contract_rs.wasm` which was recompiled
that's why I've updated binary vectors in `test_verify_chunk_invalid_state_challenge`